database/youtube/Kubernetes.json

```
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "Edureka Kubernetes Tutorial",
+        "description": "Edureka's Kubernetes YouTube tutorial offers a comprehensive and beginner-friendly guide to understanding and mastering Kubernetes, a powerful container orchestration platform.",
```

[proofreading] You can make this a bit shorter. 
```suggestion
        "description": "Edureka's Kubernetes YouTube tutorial offers a comprehensive and beginner-friendly guide to understanding and mastering Kubernetes.",
```